source/boards/work-with-boards.rst

```diff
@@ -77,7 +81,7 @@ Custom templates are fully editable, but standard templates cannot be edited or
 From Mattermost Boards v7.2 or later
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Only admins and editors of a custom template can edit the template. If you do not see the Pencil icon when hovering over the template, then you do not have the appropriate permissions to edit the template.
+Only admins and editors of a custom template can edit the template. If you don't see the pencil icon when hovering over the template, then you don't have the appropriate permissions to edit the template.
```

Comment:
A good place to introduce the Pencil icon image using the same approach as the Gear and Options icons